impl/src/main/java/com/sun/faces/util/Util.java

```diff
@@ -118,6 +119,9 @@ public class Util {
     // Log instance for this class
     private static final Logger LOGGER = FacesLogger.APPLICATION.getLogger();
 
+    public static final String EMPTY_STRING = "";
+    public static final String SPACE_STRING = " ";
```

Comment:
Should go in `RIConstants`. The `EMPTY_STRING` is already represented by `NO_VALUE`.